src/main/resources/db/changelog/007-create-product-image-table.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd">
+    <changeSet id="007-create-ordered-product" author="Ivan Aksenov">
+        <createTable tableName="product_image">
+            <column name="s3_id" type="uuid">
```

Comment:
<preConditions onFail="MARK_RAN">
            <not>
                <tableExists tableName="product_image"/>
            </not>
        </preConditions>